packages/app-project/src/shared/components/WorkflowSelector/components/WorkflowSelectButtons/WorkflowSelectButtons.js

```diff
@@ -0,0 +1,106 @@
+import { Box } from 'grommet'
+import { SpacedText } from '@zooniverse/react-components'
+import WorkflowSelectButton from '../WorkflowSelectButton'
+import en from './locales/en'
+import counterpart from 'counterpart'
+import PropTypes from 'prop-types'
+
+counterpart.registerTranslations('en', en)
+
+export default function WorkflowSelectButtons ({ assignedWorkflowID = '', onSelect, workflows = [] }) {
+  let assignedWorkflow
```

Comment:
I think a new user, or user without workflow ID assigned, on a project with the experimental feature "workflow assignment" we still want in this first block where workflow buttons are shown in unlocked and locked sections (with only first level workflow unlocked), but currently that scenario is going to the second block and showing all workflow buttons enabled.